client/payments/data/test/stub.ts

```diff
@@ -0,0 +1,50 @@
+export const paymentGatewaysStub = [
+	{
+		id: 'bacs',
+		title: 'direct bank',
+		description: 'description',
+		order: '',
+		enabled: false,
+		method_title: 'Direct bank transfer',
+		method_description: 'method description',
+		method_supports: [ 'products' ],
+		settings: {
+			title: {
+				id: 'title',
+				label: 'Title',
+				description:
+					'This controls the title which the user sees during checkout.',
+				type: 'text',
+				value: 'direct bank',
+				default: 'Direct bank transfer',
+				tip:
+					'This controls the title which the user sees during checkout.',
+				placeholder: '',
```

Comment:
I think we should create a type for this and use it https://github.com/woocommerce/woocommerce-admin/blob/8d8f1c816f2ee6c7cd7a06f2368265fa6ea49422/client/payments/data/types.ts#L10 as it appears these variables are mostly consistent.
You could call it `SettingDefinition` and use it in types as `Record<string, SettingDefinition>`, that way the setting key can still be generic.